mbs_results/ratio_of_means.py

```diff
@@ -300,6 +304,10 @@ def ratio_of_means(
             .pipe(wrap_calculate_imputation_link, **default_columns)
         )
 
+    if manual_constructions is not None:
```

Comment:
Loved that extending the ratio of means was straightforward and clear, it is a good indication that our work follows good practises